grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ReasoningTests.java

```diff
@@ -46,7 +48,7 @@ public void testSet10() {
         QueryBuilder iqb = testSet10.graph().graql().infer(true);
```

Comment:
Each test case should still have a functional purpose. What was the purpose of Szymon writing this test? What exactly was he checking. If it is a test for testing sake then I am not sure it should be added as a unit test. It should noted to become an acceptance test or release test.